xcdat/dataset.py

```diff
@@ -261,39 +345,62 @@ def decode_time_units(dataset: xr.Dataset):
     return dataset
 
 
-def keep_var(dataset: xr.Dataset, var: str) -> xr.Dataset:
-    """Keep a specific variable in the Dataset and drop the rest.
+def get_inferred_var(dataset: xr.Dataset) -> xr.DataArray:
+    """Gets the inferred data variable that is tagged in the Dataset.
+
+    This function looks for the "xcdat_infer" attribute pointing
+    to the desired data var in the Dataset, which can be set through
+    ``xcdat.open_dataset()``, ``xcdat.open_mf_dataset()``, or manually.
 
-    This function is useful for subsetting a Dataset with a large number of
-    variables that aren't of interest, which could otherwise hinder performance
-    in Dataset operations.
+    This utility function is useful for designing XCDAT APIs with an optional
+    ``data_var`` kwarg to allow implicit (``data_var=None``, calls this
+    function) or explicit (``data_var='a_var'``) selection of a data var to
+    perform the operation on.
```

Comment:
change to:

This utility function is useful for designing XCDAT APIs with an optional
``data_var`` kwarg. If ``data_var`` is None, an inference to the desired data var is performed
 with a call to this function. Otherwise, perform the API operation explicitly on ``data_var``.